hazelcast/src/test/java/com/hazelcast/internal/partition/GracefulShutdownTest.java

```diff
@@ -47,7 +47,17 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 
-@RunWith(HazelcastParallelClassRunner.class)
+/**
+ * This test creates a massive amount of threads (peaks of 1000 threads and 800 daemon threads).
+ * As comparison the {@link com.hazelcast.map.BasicMapTest} creates about 700 threads and 25 daemon threads.
+ * This regularly results in test failures when multiple PR builders run in parallel due to a resource starvation.
+ * <p>
+ * Countermeasures are to remove the {@link ParallelTest} annotation or to use the {@link HazelcastSerialClassRunner}.
+ * Without {@link ParallelTest} we'll add the whole test duration to the PR builder time (about 25 seconds).
+ * With the serial runner the test takes over 3 minutes, but with a maximum of 200 threads and 160 daemon threads.
+ * This should have less impact on other tests and the total run duration.
+ */
```

Comment:
This is not a class javadoc but a comment on `@RunWith(HazelcastSerialClassRunner.class)`. So, I suggest to use block comment here; 
```
/*
 * [...]
 */
```

Reply:
Changed to block comment.